test/solidity/Facets/DeBridgeFacet.t.sol

```diff
@@ -76,167 +71,61 @@ contract DeBridgeFacetTest is DSTest, DiamondTest {
+        bridgeData.bridge = "debridge";
+        bridgeData.minAmount = defaultUSDCAmount;
+
+        IDeBridgeGate.ChainSupportInfo memory chainConfig = IDeBridgeGate(DEBRIDGE_GATE).getChainToConfig(DST_CHAIN_ID);
+        uint256 nativeFee = addToMessageValue = chainConfig.fixedNativeFee == 0
             ? IDeBridgeGate(DEBRIDGE_GATE).globalFixedNativeFee()
             : chainConfig.fixedNativeFee;
-        executionFee = 10**usdc.decimals();
+        uint256 executionFee = 1 * 10**usdc.decimals();
 
-        validDeBridgeData = DeBridgeFacet.DeBridgeData(
+        deBridgeData = DeBridgeFacet.DeBridgeData(
             "",
-            false,
             nativeFee,
+            false,
             0,
             DeBridgeFacet.SubmissionAutoParamsTo(
                 executionFee,
                 REVERT_IF_EXTERNAL_FAIL,
-                abi.encodePacked(DAI_HOLDER),
+                abi.encodePacked(USER_RECEIVER),
                 ""
             )
         );
     }
 
-    function testRevertToBridgeTokensWhenSendingAmountIsZero() public {
-        vm.startPrank(USDC_HOLDER);
-
-        usdc.approve(address(deBridgeFacet), 10_000 * 10**usdc.decimals());
-
-        ILiFi.BridgeData memory bridgeData = validBridgeData;
-        bridgeData.minAmount = 0;
-
-        vm.expectRevert(InvalidAmount.selector);
-        deBridgeFacet.startBridgeTokensViaDeBridge(bridgeData, validDeBridgeData);
-
-        vm.stopPrank();
-    }
-
-    function testRevertToBridgeTokensWhenReceiverIsZeroAddress() public {
-        vm.startPrank(USDC_HOLDER);
-
-        usdc.approve(address(deBridgeFacet), 10_000 * 10**usdc.decimals());
-
-        ILiFi.BridgeData memory bridgeData = validBridgeData;
-        bridgeData.receiver = address(0);
-
-        vm.expectRevert(InvalidReceiver.selector);
-        deBridgeFacet.startBridgeTokensViaDeBridge(bridgeData, validDeBridgeData);
-
-        vm.stopPrank();
-    }
-
-    function testRevertToBridgeTokensWhenSenderHasNoEnoughAmount() public {
-        vm.startPrank(USDC_HOLDER);
-
-        usdc.approve(address(deBridgeFacet), 10_000 * 10**usdc.decimals());
-
-        usdc.transfer(DAI_HOLDER, usdc.balanceOf(USDC_HOLDER));
-
-        vm.expectRevert(abi.encodeWithSelector(InsufficientBalance.selector, 10 * 10**usdc.decimals(), 0));
-        deBridgeFacet.startBridgeTokensViaDeBridge(validBridgeData, validDeBridgeData);
-
-        vm.stopPrank();
-    }
-
-    function testRevertToBridgeTokensWhenSendingNoEnoughNativeAsset() public {
-        vm.startPrank(USDC_HOLDER);
-
-        ILiFi.BridgeData memory bridgeData = validBridgeData;
-        bridgeData.sendingAssetId = address(0);
-        bridgeData.minAmount = 3e18;
-
-        vm.expectRevert(InvalidAmount.selector);
-        deBridgeFacet.startBridgeTokensViaDeBridge{ value: 2e18 }(bridgeData, validDeBridgeData);
-
-        vm.stopPrank();
+    function testBase_Revert_BridgeWithInvalidDestinationCallFlag() public override {
```

Comment:
Even if the facet supports destination calls, this test should still pass. 
In Stargate and NXTP facet we have checks that make sure that if _bridgeData.hasDestinationCall is true then there must also be callData

We can discuss to not do these checks but in order to have all facets behave similar, I would recommend to add such a check and make sure the test passes


Please review the test in TestBaseFacet.sol

Reply:
updated the DeBridge facet with destination call flag validation and updated the foundry test